consensus/state.go

```diff
@@ -1939,7 +1953,8 @@ func (cs *State) addProposalBlockPart(msg *BlockPartMessage, peerID p2p.NodeID)
 
```

Comment:
Should we rename `cs.ValidBlock` to `cs.TwoThirdPrevoteBlock`, to indicate that its just about whether there are two/thirds prevotes, and not about Tendermint / App-determined validity of the block.

Reply:
I think this is a great idea. The naming should at least communicate if this is a block pre-finalization or after.